{"name": "test_get_order_by_track_invalid_track", "status": "passed", "steps": [{"name": "Проверяем, что несуществующий трек приводит к ошибке", "status": "passed", "start": 1745242255855, "stop": 1745242255855}], "start": 1745242254588, "stop": 1745242255856, "uuid": "27dcb167-1165-4950-8b78-337debb2a7e6", "historyId": "8c501d5bc66611283fab5a3ba61068c7", "testCaseId": "8c501d5bc66611283fab5a3ba61068c7", "fullName": "tests.test_additional.TestGetOrderByTrack#test_get_order_by_track_invalid_track", "labels": [{"name": "story", "value": "Негативный сценарий – запрос с несуществующим треком"}, {"name": "feature", "value": "Получение заказа по треку"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_additional"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "MacBook-Air-Eduard.local"}, {"name": "thread", "value": "15791-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_additional"}]}